main.py

```diff
@@ -327,19 +327,19 @@ class Inputs(BaseModel):
     cut_off: datetime
     timestamp_to_use: TimestampType
     account_type: AccountType
-    org_name: str | None
+    org_name: str | None = None
```

Comment:
I optional type signatures was one of the things pydantic v2 changed, but are you sure this change is needed in this case? We never rely on the default anywhere, right?

Reply:
```
$ cat example.py 
from pydantic import BaseModel
from typing import Optional

class Example(BaseModel):
    a: int
    b: int = 1
    c: Optional[str]

print(Example(b=10))
$ python3 example.py 
Traceback (most recent call last):
  File "./example.py", line 9, in <module>
    print(Example(b=10))
          ^^^^^^^^^^^^^
  File "./.venv/lib64/python3.11/site-packages/pydantic/main.py", line 164, in __init__
    __pydantic_self__.__pydantic_validator__.validate_python(data, self_instance=__pydantic_self__)
pydantic_core._pydantic_core.ValidationError: 2 validation errors for Example
a
  Field required [type=missing, input_value={'b': 10}, input_type=dict]
    For further information visit https://errors.pydantic.dev/2.4/v/missing
c
  Field required [type=missing, input_value={'b': 10}, input_type=dict]
    For further information visit https://errors.pydantic.dev/2.4/v/missing
```
pydantic2 cannot handle Optional fields without default value.
Also, if we look at definition of Optional:
```
@_SpecialForm
def Optional(self, parameters):
    """Optional[X] is equivalent to Union[X, None]."""
    arg = _type_check(parameters, f"{self} requires a single type.")
    return Union[arg, type(None)]
```
it WILL default to None, so we effectively changing nothing here.

Reply:
Yeah makes sense to me!